diff --git a/node_modules/@livekit/react-native/android/build.gradle b/node_modules/@livekit/react-native/android/build.gradle
index d1d892d..bb4d381 100644
--- a/node_modules/@livekit/react-native/android/build.gradle
+++ b/node_modules/@livekit/react-native/android/build.gradle
@@ -3,12 +3,13 @@ buildscript {
   def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['LivekitReactNative_kotlinVersion']
 
   repositories {
+    mavenLocal()
     google()
     mavenCentral()
   }
 
   dependencies {
-    classpath("com.android.tools.build:gradle:7.3.1")
+    classpath("com.android.tools.build:gradle:8.5.2")
     // noinspection DifferentKotlinGradleVersion
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
   }
@@ -51,6 +52,7 @@ android {
 }
 
 repositories {
+  mavenLocal()
   mavenCentral()
   google()
   maven { url 'https://jitpack.io' }
