diff --git a/node_modules/lottie-react-native/android/build.gradle b/node_modules/lottie-react-native/android/build.gradle
index a0dc14f..9167f4a 100644
--- a/node_modules/lottie-react-native/android/build.gradle
+++ b/node_modules/lottie-react-native/android/build.gradle
@@ -2,13 +2,14 @@ buildscript {
     def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['lottiereactnative_kotlinVersion']
 
     repositories {
+        mavenLocal()
         google()
         gradlePluginPortal()
         mavenCentral()
     }
 
     dependencies {
-        classpath("com.android.tools.build:gradle:7.2.2")
+        classpath("com.android.tools.build:gradle:8.5.2")
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
     }
 }
