diff --git a/node_modules/react-native-localize/android/build.gradle b/node_modules/react-native-localize/android/build.gradle
index f6281a5..d70112f 100644
--- a/node_modules/react-native-localize/android/build.gradle
+++ b/node_modules/react-native-localize/android/build.gradle
@@ -3,12 +3,14 @@ buildscript {
         rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
     }
     repositories {
+        mavenLocal()
         google()
         gradlePluginPortal()
+        mavenCentral()
     }
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet("kotlinVersion", "1.9.24")}")
-        classpath("com.android.tools.build:gradle:8.13.0")
+        classpath("com.android.tools.build:gradle:8.5.2")
     }
 }
 
